leapp/utils/output.py

```diff
@@ -1,10 +1,17 @@
 from __future__ import print_function
+
 import json
+import os
 import sys
 from contextlib import contextmanager
 
-from leapp.models import ErrorModel
 from leapp.exceptions import LeappRuntimeError
+from leapp.models import ErrorModel
+
+
+def _is_verbose():
+    """Redefinition of is_verbose from leapp.libraries.stdlib.config because it leads to import errors"""
+    return os.getenv('LEAPP_DEBUG', '0') == '1' or os.getenv('LEAPP_VERBOSE', '0') == '1'
```

Comment:
You don't need to check both variables. When `LEAPP_DEBUG` is set to 1, `LEAPP_VERBOSE` gets set too.
```suggestion
    return os.getenv('LEAPP_VERBOSE', '0') == '1'
```

Reply:
@drehak it's a copy and paste of the original implementation

Reply:
I see. Still, debug also sets verbose in cli code now.